Dockerizing enviroment

pedroandrade opened the pull request:
- add Dockerfile to each application
- outputs will be written in tmp volume

docker-compose will generate 4 containers:
- redis
- crystal_app
- crystal_web_app
- ruby_app

Each container will generate output on tmp volume

pedroandrade committed:
Dockerizing enviroment
- add Dockerfile to each application
- outputs will be written in tmp volume

andersondias commented on ruby-app/Dockerfile:
```diff
@@ -0,0 +1,17 @@
+
+USER nobody
+
+WORKDIR /app  
```
Please add an ENTER here and remove extra spaces.

andersondias commented on README.md:
````diff
@@ -22,34 +22,52 @@ Also, it writes some output into tmp/output.txt file:
 ```
 
 ## Setup
+You can run the application using docker:
````
Please, add a new line here.

andersondias commented on README.md:
```diff
@@ -22,34 +22,52 @@ Also, it writes some output into tmp/output.txt file:
+- ruby_app
 
-The last one is going to print out every new line written into tmp/output.txt.
+you should use command `tail -f tmp/output.txt` to see every new line writed on tmp/output.txt
```
Suggestion: "You may use `tail -f tmp/output.txt` to follow every new line written on the output file."

pedroandrade committed:
Add new line and remove extra spaces in files
- remove extra spaces
- add new lines at the end of file in Dockerfiles
- improve english in README file
